web_ui/src/pages/camera-page/components/sidebar/setting-option.component.tsx

```diff
@@ -30,17 +31,22 @@ export const SettingOption = ({ label, config, onChange }: SettingOptionProps) =
     return (
         <>
             <Flex
-                marginTop={'size-200'}
+                marginTop={'size-50'}
```

Comment:
[Nitpick] Since all these margins are the same we could use `marginStart` from line 23